Makefile

```diff
@@ -25,8 +25,11 @@ UID = $(shell id -u)
 GID = $(shell id -g)
 
 # The below files are needed and can be downloaded from the internet
-testdata_objects = testdata/vmlinux testdata/root-drive.img testdata/firecracker testdata/jailer
-firecracker_version = v0.23.0
+firecracker_version=v0.24.2
+arch=$(shell uname -m)
+# The below files are needed and can be downloaded from the internet
```

Comment:
This comment is duplicated from line 27.

Reply:
Fixed it.